### ISSUE TITLE
[2.6-beta2] Inconsistent audio settings with Fullaudio.

### ISSUE THREAD

**Issue by einhirn** [bigbluebutton/bigbluebutton#16256]
We participated in @schrd's test run of BBB 2.6-beta2 last week in the German BBB-Community (Matrix channel `BigBlueButton-DE`) and thought two config items concerning the fullaudio bridge configuration to be inconsistent.

**Describe the bug(ish)**
1. fullaudio is default in webrtc SFU config, but not in HTML5 config

> Is it intentionally still using sipjs as `defaultFullAudioBridge` in https://github.com/bigbluebutton/bigbluebutton/blob/v2.6.x-release/bigbluebutton-html5/private/config/settings.yml#L554? I browsed issues and PRs to find #14020, #14218 et al. - do these describe the reason why it's configured this way?

2. there are different parameters used for opus codec in different places.
>With PR https://github.com/bigbluebutton/bigbluebutton/pull/15766 a better audio configuration in FreeSWITCH was applied, but the parameters in the Webrtc-SFU were not modified.
For example, the fullaudio bridge sets a bitrate of 30000 and FreeSWITCH a bitrate of 64000.

**Expected behavior**
1. we expected `defaultFullAudioBridge` to be set to `fullaudio`
2. we expected the settings for WebRTC-SFU to be identical to those for FreeSWITCH

**Comment by prlanzarin** [comment]
> fullaudio is default in webrtc SFU config, but not in HTML5 config

This is not inconsistent. Those are two separate configurations with different semantics and different goals.
ie: server sides means enabling the full audio module, second one means that a _client_ (in this case: bbb-html5) should use that module. Having the module enabled by default in the server side _does not_ imply that clients should use it by default. That's not our intention for 2.6.

> there are different parameters used for opus codec in different places.

Correct - there are two ways to fix this:
  - via documentation (specify that the SFU codec parameters must be changed as well) - and our side by guaranteeing that they're consistent.
  - automatically (inherit specifications from FreeSWITCH's answer, internally

Second one is not _that_ simple and I don't see it on the top of my issue stack.

**Comment by einhirn** [comment]
> Having the module enabled by default in the server side does not imply that clients should use it by default. That's not our intention for 2.6.
Ah - thanks for clarification. Somehow this got oversimplified to "Fullaudio by default in 2.6" and thus the expectation that it would be enabled for clients by default.

> [ this can be fixed ] via documentation (specify that the SFU codec parameters must be changed as well) - and our side by guaranteeing that they're consistent.

This seems sufficient - even though the other option would be ideal...

**Comment by prlanzarin** [comment]
Yeah. I'll set this as 2.7 for the time being which is where it makes the most sense. With a 2.7 milestone we can target the ideal solution.

**Comment by prlanzarin** [comment]
Closing in favor of #21059
